src/kuzzle.js

```diff
@@ -428,6 +428,32 @@ Kuzzle.prototype.logout = function (cb) {
 };
 
 /**
+ * Checks wether a given jwt token still represents a valid session in Kuzzle.
+ *
+ * @param  {string}   token     The jwt token to check
+ * @param  {function} callback  The callback to be called when the response is
+ *                              available. The signature is `function(error, response)`.
+ * @return {Kuzzle}             The Kuzzle instance to enable chaining.
+ */
+Kuzzle.prototype.checkToken = function (token, callback) {
+  var
+    self = this,
+    request = {
+      body: {
+        token: token
+      }
+    };
+
+  if (typeof callback !== 'function') {
```

Comment:
Use `this.callbackRequired('Kuzzle.checkToken', callback)` instead.
The function `callbackRequired` is a small helper function meant to standardize error messages and reduce the number of lines of code.